app/libtest.sh

```diff
@@ -256,7 +258,20 @@ setupConode(){
 runCoBG(){
     for nb in "$@"; do
     	testOut "starting conode-server #$nb"
-    	( ./conode -d $DBG_SRV -c co$nb/private.toml | tee $COLOG$nb.log & )
+    	(
```

Comment:
The changes in this area are to make the tests explain themselves better when something goes wrong. Instead of just hoping the server is up and running, we check to see if it died on start.